src/Expanders.jl

```diff
@@ -537,9 +537,10 @@ function Selectors.runner(::Type{ExampleBlocks}, x, page, doc)
         for (ex, str) in Utilities.parseblock(code, doc, page; keywords = false)
             (value, success, backtrace, text) = Utilities.withoutput() do
                 cd(dirname(page.build)) do
-                    Core.eval(mod, Expr(:(=), :ans, ex))
+                    Core.eval(mod, ex)
                 end
             end
+            Core.eval(mod, Expr(:global, Expr(:(=), :ans, QuoteNode(value))))
```

Comment:
It's a bit weird that we set `ans` for `@example` blocks but it was tested so...

Reply:
:man_shrugging: I guess we consider every line in an at-example to be a REPL line, which is not necessarily unreasonable -- people will most likely paste it into the REPL or run it in Juno or something.

Reply:
Right, but we are not even doing it for script-style jldoctest.